packages/design-system/src/Text/Text.styles.tsx

```diff
@@ -135,7 +135,7 @@ export const StyledText = styled.span<{
 
   font-size: var(--font-size);
   font-weight: var(--font-weight);
-  line-height: var(--line-height);
```

Comment:
I don't think this makes very much sense - a lot of other components are dependent on this property for their height.

Reply:
Since vinay is also aligned to this we can do... If you see any specific component please let me know